label_sync/labels.yaml

```diff
@@ -0,0 +1,355 @@
+#  - color: fbca04
+#    name: kind/upgrade-test-failure
+  - color: 15dd18
+    name: lgtm
+  - color: ededed
+    name: needs-ok-to-merge
+  - color: b60205
+    name: needs-ok-to-test
+  - color: BDBDBD
+    name: needs-rebase
+  - color: ededed
+    name: needs-sig
+  - color: 0e8a16
+    name: non-release-blocker
+  - color: fbca04
+    name: ok-to-merge
+#  - color: fef2c0
+#    name: priority/awaiting-more-evidence
+#  - color: fbca04
+#    name: priority/backlog
+#  - color: e11d21
+#    name: priority/critical-urgent
+#  - color: e11d21
+#    name: priority/failing-test
+#  - color: eb6420
+#    name: priority/important-longterm
+#  - color: eb6420
+#    name: priority/important-soon
+#  - color: ff0000
+#    name: priority/P0
+#  - color: ededed
+#    name: priority/P1
+#  - color: ededed
+#    name: priority/P2
+#  - color: ededed
+#    name: priority/P3
+#  - color: ffaa00
+    name: queue/blocks-others
+  - color: ffaa00
+    name: queue/critical-fix
+  - color: ffaa00
+    name: queue/fix
+  - color: ffaa00
+    name: queue/multiple-rebases
+  - color: d93f0b
+    name: release-blocker
+  - color: c2e0c6
+    name: release-note
+  - color: c2e0c6
+    name: release-note-action-required
+  - color: db5a64
+    name: release-note-label-needed
+  - color: c2e0c6
+    name: release-note-none
+  - color: d93f0b
+    name: requires-release-czar-attention
+  - color: eb6420
+    name: retest-not-required
+  - color: fbca04
+    name: retest-not-required-docs-only
+  - color: d2b48c
+    name: sig/api-machinery
+  - color: d2b48c
+    name: sig/apps
+  - color: d2b48c
+    name: sig/architecture
+  - color: d2b48c
+    name: sig/auth
+  - color: d2b48c
+    name: sig/autoscaling
+  - color: d2b48c
+    name: sig/aws
+  - color: d2b48c
+    name: sig/azure
+  - color: d2b48c
+    name: sig/big-data
+  - color: d2b48c
+    name: sig/cli
+  - color: d2b48c
+    name: sig/cluster-lifecycle
+  - color: d2b48c
+    name: sig/cluster-ops
+  - color: d2b48c
+    name: sig/contributor-experience
+  - color: d2b48c
+    name: sig/docs
+  - color: d2b48c
+    name: sig/federation
+  - color: d2b48c
+    name: sig/instrumentation
+  - color: d2b48c
+    name: sig/network
+  - color: d2b48c
+    name: sig/node
+  - color: d2b48c
+    name: sig/onprem
+  - color: d2b48c
+    name: sig/openstack
+  - color: d2b48c
+    name: sig/release
+  - color: d2b48c
+    name: sig/rktnetes
+  - color: d2b48c
+    name: sig/scalability
+  - color: d2b48c
+    name: sig/scheduling
+  - color: d2b48c
+    name: sig/service-catalog
+  - color: d2b48c
+    name: sig/storage
+  - color: d2b48c
+    name: sig/testing
+  - color: d2b48c
+    name: sig/ui
+  - color: d2b48c
+    name: sig/windows
+  - color: ee9900
+    name: size/L
+  - color: eebb00
+    name: size/M
+  - color: 77bb00
+    name: size/S
+  - color: ee5500
+    name: size/XL
+  - color: "009900"
+    name: size/XS
+  - color: ee0000
+    name: size/XXL
+  - color: "795548"
+    name: stale
```

Comment:
Addressed all labels related feedback. I didn't removed any labels - I'm only commenting or uncommenting - I want to have full labels set for further/final discussion - so any label that should be removed is kept commented out atm.